.markdownlint.yml

```diff
@@ -0,0 +1,7 @@
+default: true
```

Comment:
Introducing this file to make it easier to locally enforce markdown styles.
I'm not suggesting that we should require a particular style guide for this repo, but it'll make it easier for anyone who want so opt in to using [markdownlint](https://github.com/DavidAnson/markdownlint) in their IDE.